config/initializers/beyondz.rb

```diff
@@ -18,6 +18,16 @@ def self.bitly_access_token
     @config[:bitly_access_token]
   end
 
+  def self.production?
+    @config[:base_url] == 'https://join.bebraven.org/'
```

Comment:
On Thu, Feb 25, 2016 at 12:23:27PM -0800, Brian Sadler wrote:
> > +  def self.production?
> > +    @config[:base_url] == 'https://join.bebraven.org/'
> 
> Was this committed accidentally?

No, I copied the url off the config for production.